### PR TITLE
Moving Import Local Mirabuf

### PR DESCRIPTION
**Pull request by Dhruv-0-Arora**
### Description
Removed the "Import Local Mirabuf" from MainHUD and moved it to "Spawn Asset" panel.

Screenshot 2024-07-31 at 1 19 34 PM

Screenshot 2024-07-31 at 1 20 25 PM

> **Note:** Merge #1069 first

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1787)

### CODE REVIEW

**Review by a-crowell — CHANGES_REQUESTED** [review]
Looking good! Make sure the control schemes panel still pops up upon uploading a robot.

**Review by a-crowell — APPROVED** [review]
LGTM!

File: fission/src/ui/components/MainHUD.tsx
Comment (BrandonPacewic) [inline fission/src/ui/components/MainHUD.tsx]: Thinking we should remove this section now.

**Review by BrandonPacewic — COMMENTED** [review]
This helps with one of our pain points from beta day about `mirabuf` being confusing. Having everything under spawn asset is definitely better imo.

**Comment by LucaHaverty** [comment]
Do you all think it looks better for the button to be at the bottom of the panel instead of under "saved assets"? I think it looks a bit cleaner at the bottom, but here's a comparison:

Under saved assets             |  Bottom
:-------------------------:|:-------------------------:
image  |  image

Also, maybe we could add an icon to the button kind of like this:
image

File: fission/src/ui/components/MainHUD.tsx
Comment (Dhruv-0-Arora) [reply fission/src/ui/components/MainHUD.tsx]: Whoops

**Review by BrandonPacewic — COMMENTED** [review]
~~I'm ready to approve this once we decide what we want to do with Luca's suggestions.~~

https://github.com/Autodesk/synthesis/issues/1071#issuecomment-2274481787

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]
We should wait to put this in until after #1068 since it will result in a merge conflict. Should be easier to fix in a smaller PR like this.

> [!note]
> - Waiting for #1068.

**Review by LucaHaverty — APPROVED** [review]

**Review by BrandonPacewic — APPROVED** [review]

**Pull request merged**
